appveyor.yml

```diff
@@ -39,6 +39,9 @@ install:
   - 7z x scons-2.5.1.zip
   - cd scons-2.5.1
   - python setup.py install
+  - echo *** Patching NSIS
+  - cd %APPVEYOR_BUILD_FOLDER%
+  - patch "C:\Program Files (x86)\NSIS\Include\MultiUser.nsh" < build\appveyor\NSIS.patch
```

Comment:
If this fails the build fails, right? Just worried about if they ever install a 64-bit version of NSIS (does that even exist?).